chunks/tool-chrome/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG base
+FROM ${base}
+
+USER gitpod
+
+# Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
+ENV TRIGGER_REBUILD=1
+
+# chrome and basic render font
+RUN apt-get update \
+  && apt-get install -y apt-transport-https \
+  && curl -sSL https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add - \
+  && echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list \
+  && apt-get update && apt-get install -y google-chrome-stable --no-install-recommends \
+  && apt-get install -y fonts-noto fonts-noto-cjk 
+
+# misc deps for electron and puppeteer to run
+RUN sudo apt-get update \
+ && sudo apt-get install -y \
+  libasound2-dev \
+  libgtk-3-dev \
+  libnss3-dev
```

Comment:
@gengjiawen may I ask you to share references for these dependencies? I ask because a searched some results, the the libraries needed appear to vary. [[1](https://postsrc.com/code-snippets/how-to-install-latest-stable-chrome-version-in-ubuntu-puppeteer)][[2](https://www.electronjs.org/docs/latest/development/build-instructions-linux)]

If you were able to install these manually and get them working from our current node image, perhaps you could share a short Loom video?

Reply:
This is what I find when I use gitpod for electron dev: see https://github.com/gengjiawen/electron-devdocs/blob/1937127c45a8d138400f68b3022b4280170bf6db/.gitpod.Dockerfile#L3.